src/common/constants/organization.js

```diff
@@ -15,7 +15,18 @@ export const ORGANIZATION_STATUS_LABELS = {
 
 export const ORGANIZATION_PAYMENT_METHOD = {
   cash: 'Cash',
-  card: 'Credit Card/P-Card',
+  card: 'Credit Card',
+  eft: 'EFT',
+  depAccount: 'Deposit Account',
+  physicalCheck: 'Physical Check',
+  bankDraft: 'Bank Draft',
+  internalTransfer: 'Internal Transfer',
+  other: 'Other',
+};
+
+export const ORGANIZATION_PAYMENT_METHOD_FOR_FILTER = {
+  cash: 'Cash',
```

Comment:
If this is a subset of the full payment methods list please use constants, don't copy-paste string values.